native/cocos/core/scene-graph/Node.cpp

```diff
@@ -886,6 +886,16 @@ void Node::clearNodeArray() {
     }
 }
 
+ccstd::string Node::getPathInHierarchy() const {
+    ccstd::string result = getName();
+    Node *curNode = getParent();
+    while (curNode && dynamic_cast<Scene *>(curNode) == nullptr) {
+        result = ccstd::string(curNode->getName()).append("/").append(result);
```

Comment:
can reuse result, just insert contents ahead of it, such as
```c++
result.insert(0, "/").insert(0, curNode->getName());
```

Reply:
It's a good idea!